sound/soc/intel/boards/sof_sdw.c

```diff
@@ -236,6 +236,17 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					SOF_SDW_PCH_DMIC |
 					RT711_JD2_100K),
 	},
+	{
+		/* NUC15 LAPRC710 skews */
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_BOARD_VENDOR, "Intel Corporation"),
+			DMI_MATCH(DMI_BOARD_NAME, "LAPRC710"),
+		},
+		.driver_data = (void *)(SOF_SDW_TGL_HDMI |
+					SOF_SDW_PCH_DMIC |
+					RT711_JD2_100K),
+	},
```

Comment:
Can we remove the `/* NUC15 'Rooks County' LAPRC510 and LAPRC710 skews */` entry if this is more generic?

Reply:
I'd say no. My suggestion is more generic, but handles only the LAPRC710 version (genuine Intel and rebranded variants as well). The entry above my suggestion handles both LAPRC510 and LAPRC710 versions, but only the genuine Intel variants. As much as I know, the LAPRC510 wasn't rebranded and marketed that way, so it doesn't need to be handled "in a generic" way. I think if we leave the code in this state it handles all the cases.